pkg/kubelet/kubelet.go

```diff
@@ -1722,12 +1752,41 @@ func (kl *Kubelet) makePodDataDirs(pod *api.Pod) error {
 	return nil
 }
 
+// syncPod is the transaction script for the sync of a single pod.
+//
+// Arguments:
+//
+// pod - the pod to sync
+// mirrorPod - the mirror pod for the pod to sync, if it is a static pod
+// podStatus - the current status (TODO: always from the status manager?)
+// updateType - the type of update (CREATE,ADD,DELETE,RECONCILE)
```

Comment:
The types should be ADD, UPDATE, REMOVE, RECONCILE

DELETE is handled as part of UPDATE. We should probably surface DELETE as a separate type for clarity.